lra/coordinator/server/Dockerfile

```diff
@@ -14,28 +14,34 @@
 # limitations under the License.
 #
 
-FROM maven:3.6.3-openjdk-17-slim as build
+FROM container-registry.oracle.com/java/openjdk:17 as build
```

Comment:
`container-registry.oracle.com/java/jdk-no-fee-term:17` will be a newer version and should still be freely available (no login, no license acceptance). Might want to give that a try.